sample/report/report_sample.json

```diff
@@ -0,0 +1,20 @@
+{
+  "id": "<unique-id>",
+  "name": "<name>",
+  "type": "<type>", // sql, key-value, parameter
+  "reportFile": "/private/...",
+  "configuration": {
+    "connection": "connection",
+    "provider": "<ado.net provider>",
+
+    "isListBaed": false, // only for key-value
+    "parameter": [ // only for key-value
```

Comment:
is this supposed to be // only for parameter?

Reply:
the "parameter" field is important for parameter-based and key-value reports
but the "parameter"-property looks a little bit different.

Reply:
Key-Value-Report: https://github.com/simplic/simplic-framework/blob/master/src/Simplic.Reporting/Configuration/KeyValueConfigurationModel.cs#L152
Parameter-Report: https://github.com/simplic/simplic-framework/blob/master/src/Simplic.Reporting/Configuration/ParameterConfigurationModel.cs#L266 and https://github.com/simplic/simplic-framework/blob/master/src/Simplic.Reporting/Configuration/ParameterConfigurationModel.cs#L278

Reply:
Manager: https://github.com/simplic/simplic-framework/blob/master/src/Simplic.Reporting/ReportManager.cs